e2e/protractor/suites/actions/library/library-actions.test.ts

```diff
@@ -1,429 +0,0 @@
-  const sitePublic3Admin = `admin-public3-${Utils.random()}`;
-  const sitePublic4Admin = `admin-public4-${Utils.random()}`;
-  const sitePublic5Admin = `admin-public5-${Utils.random()}`;
-  const sitePublic6Admin = `admin-public6-${Utils.random()}`;
-  const sitePublic7Admin = `admin-public7-${Utils.random()}`;
-  const sitePublic8Admin = `admin-public8-${Utils.random()}`;
-
-  const sitePublicUser = `user-public1-${Utils.random()}`;
-  const siteForDelete1 = `user-public2-${Utils.random()}`;
-  const siteForDelete2 = `user-public3-${Utils.random()}`;
-
-  const siteModerated1Admin = `admin-moderated1-${Utils.random()}`;
-  const siteModerated2Admin = `admin-moderated2-${Utils.random()}`;
-
-  const siteSearchModerated1Admin = `site-moderated-search-1-${Utils.random()}`;
-  const siteSearchModerated2Admin = `site-moderated-search-2-${Utils.random()}`;
-
-  const siteSearchPublic1Admin = `site-public-search-1-${Utils.random()}`;
-  const siteSearchPublic2Admin = `site-public-search-2-${Utils.random()}`;
-  const siteSearchPublic3Admin = `site-public-search-3-${Utils.random()}`;
-  const siteSearchPublic4Admin = `site-public-search-4-${Utils.random()}`;
-  const siteSearchForDelete = `site-public-search-5-${Utils.random()}`;
-
-  const apis = {
-    user: new RepoClient(username, username)
-  };
-
-  const loginPage = new LoginPage();
-  const page = new BrowsingPage();
-  const { dataTable, toolbar } = page;
-  const { searchInput } = page.pageLayoutHeader;
-
-  const confirmDialog = new ConfirmDialog();
-  const adminApiActions = new AdminActions();
-
-  const publicAdminSites = [siteSearchPublic1Admin, siteSearchPublic2Admin, siteSearchPublic3Admin, siteSearchPublic4Admin];
-  const moderatedSites = [siteSearchModerated1Admin, siteSearchModerated2Admin];
-
-  beforeAll(async () => {
-    await adminApiActions.createUser({ username });
-
-    await adminApiActions.sites.createSites(publicAdminSites);
-    await adminApiActions.sites.createSites(moderatedSites, SITE_VISIBILITY.MODERATED);
-
-    await loginPage.loginWith(username);
-  });
-
-  beforeEach(async () => {
-    await Utils.pressEscape();
-  });
-
-  afterEach(async () => {
-    await Utils.pressEscape();
-    await page.header.expandSideNav();
-    await page.clickPersonalFiles();
-  });
-
-  afterAll(async () => {
-    await adminApiActions.login();
-    await adminApiActions.deleteSites([...publicAdminSites, ...moderatedSites]);
-  });
-
-  describe('Join a public library', () => {
-    beforeAll(async () => {
-      await adminApiActions.sites.createSite(sitePublic1Admin);
-      await apis.user.favorites.addFavoriteById('site', sitePublic1Admin);
-    });
-
-    afterAll(async () => {
-      await adminApiActions.sites.deleteSite(sitePublic1Admin);
-    });
-
-    it('[C290105] from Favorite Libraries', async () => {
-      await page.goToFavoriteLibrariesAndWait();
-      await dataTable.selectItem(sitePublic1Admin);
-      await BrowserActions.click(toolbar.joinButton);
```

Comment:
joinButton field can be removed from Toolbar class as it looks like it is unused now

Reply:
deleted in [aa865dc](https://github.com/Alfresco/alfresco-content-app/pull/3545/commits/aa865dcf243348da373c4560117ab99260fb3fe8)